extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -580,6 +583,7 @@ describe('amp-a4a', () => {
               'xhr.fetchTextAndHeaders called exactly once').to.be.true;
           expect(extractCreativeAndSignatureSpy.calledOnce,
               'extractCreativeAndSignatureSpy called exactly once').to.be.true;
+          expect(loadExtensionSpy.withArgs('amp-font').calledOnce).to.be.true;
```

Comment:
I think you can also write this as
```javascript
expect(loadExtensionSpy.withArgs('amp-font')).to.be.calledOnce;
```
You might also be able to write
```javascript
expect(loadExtensionSpy).to.be.calledOnce.withArgs('amp-font')
```
but I'm less sure of that.

Also, would be nice to have a test for `>1` and one for exactly 0 extensions.